web/saml.go

```diff
@@ -118,14 +118,12 @@ func completeSaml(c *Context, w http.ResponseWriter, r *http.Request) {
 	switch action {
 	case model.OAUTH_ACTION_SIGNUP:
 		teamId := relayProps["team_id"]
-		if len(teamId) > 0 {
-			c.App.Srv().Go(func() {
```

Comment:
@sbishel / @mkraft - Same question as earlier. I didn't see any particular cause for delays except for a few additional DB calls and websocket messages. Nothing that should cause any noticeable delay.

This is also achievable by deep copying the user and doing it in a goroutine. But I wanted to check if there was anything that I missed.

Reply:
I'm not as familiar with this section, but tracing it through, I agree. Shouldn't cause noticable delay.